delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/Event.scala

```diff
@@ -34,8 +34,20 @@ object Event {
       * @return the project where the event belongs
       */
     def project: ProjectRef
+
+  }
+
+  trait OrganizationScopedEvent extends Event {
```

Comment:
We don't really need this for `OrganizationScopedEvent` as only OrganizationEvent fits in it no ?

Reply:
Does it hurt? I wanted to have it for consistency. Now we have 3 subclasses of `Event` and each of them makes sense.

Reply:
It felt also weird when I read it as `ProjectScopedEvent` could be seen as `OrganizationScopedEvent`
Wouldn't it make more sense if you want to keep `OrganizationScopedEvent` to have that approach ?
Transform `label` to `organizationLabel` implement it in `ProjectScopedEvent` and in EventTags, `forProjectScopedEvent` would be `forOrganizationScopedEvent` merged with project related tags ?